publish.gradle

```diff
@@ -0,0 +1,56 @@
+// Configuration for Maven Central by Sonatype.
+// See: https://github.com/gradle-nexus/publish-plugin
+nexusPublishing {
+    repositories {
+        sonatype {
+            nexusUrl.set(uri('https://s01.oss.sonatype.org/service/local/'))
+            snapshotRepositoryUrl.set(uri('https://s01.oss.sonatype.org/content/repositories/snapshots/'))
+            username = System.getenv('MAVEN_USERNAME')
+            password = System.getenv('MAVEN_PASSWORD')
+        }
+    }
+}
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            from components.java
+
+            pom {
+                group = 'dev.openfga'
+                name = 'openfga-sdk'
```

Comment:
The final SDK package should be `dev.openfga.sdk`

Reply:
Hmm.. It seems this is not the recommended naming convention for Java. Let's stick to `dev.openfga:openfga-sdk`

Reply:
I think both could be fine, but I'll stick with this